app/views/shared/_flash.html.erb

```diff
@@ -0,0 +1,15 @@
+<% [:success, :info, :warning, :danger, :notice, :alert].select { |k| flash[k].present? }.each do |k| %>
```

Comment:
Yeah, I copied this from another app and I was wondering if it was better to keep them the same. But if that were the case it would be better to merge them, so let's just keep it simple here and if they merge later they merge later.

Reply:
Nice! 👍